src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,55 @@
+.nav {
+  font-weight: 500;
+
+  &__list {
+    display: flex;
+    list-style-type: none;
+    padding-left: 0;
+    margin: 0;
+  }
+
+  &__item {
+    margin: 0 5px;
+    padding: 0 5px;
+    position: relative;
+
+    &:first-child {
+      padding-left: 0;
+      margin-left: 0;
+    }
+
+    &:last-child {
+      padding-right: 0;
+      margin-right: 0;
+    }
```

Comment:
seems redundant if used `margin-left: 20px;`
```suggestion
```

Reply:
Yes, removed. Thank you.